Wordle.Api/Wordle.Api/Services/WordEditService.cs

```diff
@@ -0,0 +1,54 @@
+﻿using Microsoft.EntityFrameworkCore;
+using Wordle.Api.Dtos;
+using Wordle.Api.Models;
+
+namespace Wordle.Api.Services;
+
+public class WordEditService(WordleDbContext Db)
+{
+    public WordleDbContext Db { get; set; } = Db;
+
+    public async Task AddWord(WordDto wordToAdd)
+    {
+        if (wordToAdd.Word.Length != 5)
+        {
+            return;
+        }
+
+        Word word = new Word()
+        {
+            Text = wordToAdd.Word,
+            IsCommon = wordToAdd.IsCommon
+        };
+
+        Db.Words.Add(word);
+
+        await Db.SaveChangesAsync();
+    }
+
+    public async Task RemoveWord(string wordToRemove)
+    {
+        Word? word = await Db.Words.FirstOrDefaultAsync(word => word.Text.ToLower() == wordToRemove.ToLower());
+
+        if(word is not null)
+        {
+            Db.Words.Remove(word);
+            await Db.SaveChangesAsync();
+        }
+
+    }
+
+    public async Task UpdateWord(WordDto wordToUpdate)
+    {
+        Word? word = await Db.Words.FirstOrDefaultAsync(word => word.Text.ToLower() == wordToUpdate.Word.ToLower());
+
+        if (word is not null)
+        {
+            word.IsCommon = wordToUpdate.IsCommon;
```

Comment:
This could probably be simplified into just a toggle?
I.e, "word.IsCommon = !word.IsCommon"

Reply:
I wrote this without knowing the front end would make this a toggle instead of a setting. I like the potential flexibility this offers if we were to change the front end, but as is, you are right.